sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/QueuePropertiesExtensions.cs

```diff
@@ -167,10 +176,11 @@ private static QueueProperties ParseFromEntryElement(XElement xEntry)
             return properties;
         }
 
-        public static List<QueueProperties> ParseCollectionFromContent(string xml)
+        public static async Task<List<QueueProperties>> ParseCollectionFromContentAsync(Response response, ClientDiagnostics diagnostics)
```

Comment:
Same thing here with method name as it is no longer taking in content directly. `ParseCollectionFromResponse`? (this would apply to all versions of this method in the other extension classes)